src/main/java/core/basesyntax/impl/StorageData.java

```diff
@@ -0,0 +1,45 @@
+package core.basesyntax.impl;
+
+@SuppressWarnings("unchecked")
+public class StorageData<K, S> {
```

Comment:
```suggestion
public class StorageData<K, V> {
```
Why S?)
V - is a short for `Value`